electric/8.5.connector.py

```diff
@@ -0,0 +1,93 @@
+from electric.logic_gate import LogicGate
+
+
+class Connector:
+    """
+      Connects input gates to a gate that computes a logic operation.
+
+      Attributes:
+          gate (LogicGate): The gate that computes a logic operation.
+          input_gates (list): The input gates connected to the gate.
+
+      Methods:
+          send_inputs: Computes the output of the gate given the inputs.
+          calculate_output: Calculates the output of each input gate.
+          count_gates: Counts the number of gates in a Connector or list of Connectors.
+
+      """
+
+    def __init__(self, gate, input_gates):
+        self.gate = gate
+        self.input_gates = input_gates
+
+    def send_inputs(self, inputs):
+        """
+        Computes the output of the gate given the inputs.
+
+        Args:
+            inputs (list): The input values for the gate.
+
+        Returns:
+            The output value of the gate.
+        """
+        return self.gate.output(*self.calculate_input_gates_outputs(inputs))
+
+    def calculate_input_gates_outputs(self, inputs):
```

Comment:
Use type hints and return types -
https://www.geeksforgeeks.org/duck-typing-in-python/

Reply:
done